Frameworks/UIKit/UIButton.mm

```diff
@@ -784,17 +872,18 @@ - (UIEdgeInsets)contentEdgeInsets {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
 */
 - (void)setShowsTouchWhenHighlighted:(BOOL)showsTouch {
     UNIMPLEMENTED();
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes Xaml doesn't have built in support for showing 'glowing' touch feedback.
 */
 - (BOOL)showsTouchWhenHighlighted {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Xaml doesn't have built in support for showing 'glowing' touch feedback.");
```

Comment:
propagate this message to the setter as well; people will be calling the setter _far_ more often than the getter.